src/theme/palette.ts

```diff
@@ -1,18 +1,36 @@
+import { OriginColorVariant } from "../types"
 import { PaletteOptions } from "@mui/material"
 
 // Define custom color types
 
+export function createGradient2(color1?: string, color2?: string) {
+  console.log("color 1==?", color1, "color 2 ==>", color2)
+  if (color1 && color2)
+    return `linear-gradient(90deg, ${color1} 0%, ${color2} 100%)`
+}
+
+// function createGradient3(color1:string,color2:string,color3:string){
+
+// }
+
+type GradientsPaletteOptions = {
+  primaryLight?: string
```

Comment:
Should all of these types be optional?

Reply:
Don't need to be optional.
Actually, we defined most of gradient colors but didn't use much on design. So I removed most of theme only give `primary`, `secondary`, `gold`, `logo`.
We can give more theme colors when we'll need.